src/components/Home/Index.jsx

```diff
@@ -67,21 +67,28 @@ function Index() {
           </p>
           <p className="p-description">
             The Terasology project was born from a Minecraft-inspired tech demo
-            and is becoming a stable platform for various types of gameplay
-            settings in a voxel world. The creators and maintainers are a
-            diverse mix of software developers, designers, game testers, graphic
-            artists, and musicians. We encourage others to join!
+            in 2011. Over the years it has been a playground for different kinds
+            of technical experiments and proof of concepts. The community
+            strives to make Terasology a stable platform for various types of
+            gameplay settings in a voxel world. The contributors are a diverse
+            mix of software developers, designers, game testers, graphic
+            artists, and musicians.
           </p>
-          <div className="my-5">
-            <button
-              type="button"
-              className="font-weight-bold btn btn-lg btn-success home-btn"
-            >
-              <Link to="#about-us" className="link-about">
-                Learn More
+          <Col className="my-5 justify-content-around">
+            <Button type="button" color="primary" size="lg">
+              <Link to="#about" className="link-about">
```

Comment:
This should link to the anchor `#about-us` which is auto-generated by `<Section title="About Us"/>` from the title.

Reply:
oh I forgot about the auto-generation, will fix asap